user-interface/src/lib/hooks/UseWindowSize.ts

```diff
@@ -0,0 +1,14 @@
+import { useLayoutEffect, useState } from 'react';
+
+export default function useWindowSize() {
+  const [windowSize, setWindowSize] = useState([0, 0]);
+  useLayoutEffect(() => {
+    function updateWindowSize() {
+      setWindowSize([window.innerWidth, window.innerHeight]);
+    }
+    window.addEventListener('resize', updateWindowSize);
+    updateWindowSize();
+    return () => window.removeEventListener('resize', updateWindowSize);
+  }, []);
+  return windowSize;
+}
```

Comment:
Stash somewhere, if it's needed at some point.  It doesn't need to be committed to the repo.